apps/playground/src/app/modules/components/footer-module/footer-module.component.ts

```diff
@@ -20,40 +20,46 @@ import { Subject, takeUntil } from 'rxjs';
 interface FormFooter {
   institution: FormControl<string>;
   description: FormControl<string>;
-  displayBrands: FormControl<boolean>;
+  displayPartners: FormControl<boolean>;
   displayCategoriesLink: FormControl<boolean>;
   displayOperator: FormControl<boolean>;
 }
```

Comment:
While you're at it, and given our new way to implement Typed Forms, could you please remove this and define the form right where you're declaring the property ? ( line 276 )

Don't forget to remove the typing and let TypeScript infer everything as well as removing the definition in `initForms`